app/components/speaker-card.hbs

```diff
@@ -0,0 +1,32 @@
+<div class="session-wrapper">
+	{{#each @speakersData as |speakerDetails| }}
+		<div class="session-info">
+			<h1 class="topic-name">{{speakerDetails.topic.name}}</h1>
+			<h2 class="speaker-name">{{speakerDetails.name}}
+				{{#if speakerDetails.links.linkedin}}
+					<span class="speaker-social-info linkedin-info">
+						<a target="_blank" rel="noopener noreferrer" href={{speakerDetails.links.linkedin}}>
+							<img class="social-media-logo" src="assets/images/linkedin512.png" alt="LinkedIn logo" />
```

Comment:
NIT: Can we rename the name of the png to just "LinkedIn"?

Reply:
Any reason for this? I want to know.